plugins/main/public/react-services/format-number.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Wazuh app - Time and date functions
+ * Copyright (C) 2015-2022 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ */
+
+export function formatUINumber(number: number) {
+  if (typeof number !== 'number') {
+    return '-';
```

Comment:
Maybe if it is not a number, the value it has can be returned instead of a `-`, If it is changed, perhaps the parameter name would change